website/src/main/resources/microsite/js/addEditLinks.js

```diff
@@ -1,12 +1,16 @@
-let relativePath = window.location.pathname.replace('/scalafix/docs/', '').replace('.html', '.md');
+---
+---
```

Comment:
this is an empty YAML block, which causes to Jekyll to process this file and replace variables in the source below. Ref: https://stackoverflow.com/a/4307177/846273